mlrun/api/utils/clients/log_collector.py

```diff
@@ -26,6 +26,34 @@
 class LogCollectorErrorCode(enum.Enum):
     ErrCodeNotFound = 0
     ErrCodeInternal = 1
+    ErrCodeBadRequest = 2
+
+    @staticmethod
+    def map_error_to_exception(
+        error_code: int, error_message: str, failure_message: str
+    ) -> Exception:
+        """
+        Map error code to exception
+        :param error_code: The error code
+        :param error_message: The error message
+        :param failure_message: The failure message to use in the exception, according to the failed request
+        """
+        message = f"{failure_message},error= {error_message}"
+
+        return {
+            LogCollectorErrorCode.ErrCodeNotFound: mlrun.errors.MLRunNotFoundError(
+                message
+            ),
+            LogCollectorErrorCode.ErrCodeInternal: mlrun.errors.MLRunInternalServerError(
+                message
+            ),
+            LogCollectorErrorCode.ErrCodeBadRequest: mlrun.errors.MLRunBadRequestError(
+                message
+            ),
```

Comment:
I don't really see the point with initializing all error classes just to return one error class, I would suggest defining a static mapping between the classes and then initialize only the relevant class.